mac/arch.c

```diff
@@ -461,7 +461,7 @@ bool arch_archInit(honggfuzz_t* hfuzz) {
      */
     pthread_t exception_thread;
 
-    if (pthread_create(&exception_thread, NULL, wait_for_exception, 0)) {
+    if (pthread_create(&exception_thread, NULL, (void * _Nullable (* _Nonnull)(void * _Nullable))wait_for_exception, 0)) {
```

Comment:
I think you re "pushing" a little here :-) not sure the cast is so necessary once you fix above.